target/linux/realtek/dts-5.10/rtl838x_d-link_dgs-1210_soc.dtsi

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
+// Includes d-link dgs-1210 SoC common functions.
```

Comment:
```suggestion
/* Includes d-link dgs-1210 SoC common functions. */
```
Linus still prefers `/*` comments over `//` comments (bar the SDPX license identifier ...)

Reply:
Comment has been removed.